<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetDialog</class>
 <widget class="QDialog" name="ProjetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Projet - ArchiFlow</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>Informations Générales</string>
      </attribute>
      <layout class="QFormLayout" name="formLayoutGeneral">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelNom">
         <property name="text">
          <string>Nom du Projet *:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditNom">
         <property name="placeholderText">
          <string>Entrez le nom du projet</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDescription">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEditDescription">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Description détaillée du projet</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCategorie">
         <property name="text">
          <string>Catégorie *:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxCategorie">
         <item>
          <property name="text">
           <string>Résidentiel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Commercial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Industriel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Institutionnel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rénovation</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelStatut">
         <property name="text">
          <string>Statut:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxStatut">
         <item>
          <property name="text">
           <string>En cours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Terminé</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>En attente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Annulé</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Suspendu</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelClient">
         <property name="text">
          <string>Client:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditClient">
         <property name="placeholderText">
          <string>Nom du client</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelArchitecte">
         <property name="text">
          <string>Architecte:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditArchitecte">
         <property name="placeholderText">
          <string>Nom de l'architecte</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelProgression">
         <property name="text">
          <string>Progression:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="layoutProgression">
         <item>
          <widget class="QSlider" name="sliderProgression">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxProgression">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDetails">
      <attribute name="title">
       <string>Détails du Projet</string>
      </attribute>
      <layout class="QFormLayout" name="formLayoutDetails">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelBudget">
         <property name="text">
          <string>Budget:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxBudget">
         <property name="suffix">
          <string> €</string>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelSurface">
         <property name="text">
          <string>Surface:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSurface">
         <property name="suffix">
          <string> m²</string>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelEtage">
         <property name="text">
          <string>Nombre d'étages:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxEtage">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelMateriauPrincipal">
         <property name="text">
          <string>Matériau Principal:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxMateriauPrincipal">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Béton</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Acier</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bois</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brique</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pierre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Verre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mixte</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlanification">
      <attribute name="title">
       <string>Planification</string>
      </attribute>
      <layout class="QFormLayout" name="formLayoutPlanification">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelDateDebut">
         <property name="text">
          <string>Date de début:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEditDebut">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDateFinEstimee">
         <property name="text">
          <string>Date de fin estimée:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dateEditFinEstimee">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDureeEstimee">
         <property name="text">
          <string>Durée estimée:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelDureeCalculee">
         <property name="text">
          <string>-- jours</string>
         </property>
         <property name="styleSheet">
          <string>font-weight: bold; color: #E3C6B0;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLocalisation">
      <attribute name="title">
       <string>Localisation</string>
      </attribute>
      <layout class="QFormLayout" name="formLayoutLocalisation">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelAdresse">
         <property name="text">
          <string>Adresse:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditAdresse">
         <property name="placeholderText">
          <string>Adresse complète du projet</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLatitude">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxLatitude">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLongitude">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxLongitude">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="layoutLocationButtons">
         <item>
          <widget class="QPushButton" name="pushButtonRechercherLocalisation">
           <property name="text">
            <string>Rechercher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearLocalisation">
           <property name="text">
            <string>Effacer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameValidation">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="layoutValidation">
      <item>
       <widget class="QLabel" name="labelValidationMessage">
        <property name="text">
         <string/>
        </property>
        <property name="styleSheet">
         <string>color: #FF6B6B; font-weight: bold;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerValidation">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderProgression</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxProgression</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxProgression</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderProgression</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
